libkbfs/tlf_journal_test.go

```diff
@@ -1148,6 +1159,13 @@ func testTLFJournalPauseBlocksAndConvertBranch(t *testing.T,
 		markers++
 	}
 
+	// Wait for the local squash branch to appear.
+	select {
+	case <-branchCh:
+	case <-ctx.Done():
```

Comment:
I can't find a deadline or timeout on this ctx. If we rely on `-test.timeout`, maybe this isn't necessary?